src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -352,7 +235,7 @@ public boolean logOut() {
    *         if the user chose not to log out
    */
   public boolean logOut(boolean showPrompt) {
```

Comment:
Good point. Considering the model we put in this library, I think it makes a lot of sense to add a method to log out a single account. (However, it has been suggested that logout in Eclipse will do all-account logout — the same behavior as GMail.)

I'll rename this method to `logOutAll()` and make a note in the javadoc.